specification/http/1.0/openapi.json

```diff
@@ -0,0 +1,487 @@
+              "authorization": {
+                "type": "array",
+                "description": "The authorization information for the tool call.",
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "id": {
+                      "type": "string",
+                      "description": "The unique identifier for the authorization method or authorization provider."
+                    },
+                    "token": {
+                      "type": "string",
+                      "description": "The token for the tool call."
+                    }
+                  },
+                  "required": ["id", "token"],
+                  "additionalProperties": false
+                }
+              },
+              "secrets": {
+                "type": "array",
+                "description": "The secrets for the tool call.",
+                "items": {
+                  "$ref": "#/components/schemas/ToolSecret"
+                },
+                "additionalProperties": true
+              },
+              "user_id": {
+                "type": "string",
+                "description": "A unique ID that identifies the user, if present."
+              }
+            },
+            "additionalProperties": true
+          }
+        },
+        "required": ["tool_id"],
+        "additionalProperties": false
+      },
+      "CallToolResponse": {
+        "type": "object",
+        "properties": {
+          "call_id": {
+            "type": "string",
+            "description": "The unique identifier (e.g. UUID) for this tool call. If an ID is not provided by the client, the server will generate one."
+          },
+          "duration": {
+            "type": "number",
+            "description": "The runtime duration of the tool call, in milliseconds"
+          },
+          "success": {
+            "type": "boolean",
+            "description": "Whether the tool call was successful. If the `error` field is present, this field will be false."
+          }
+        },
+        "required": ["call_id", "success"],
+        "oneOf": [
+          {
+            "properties": {
+              "success": { "enum": [true] },
+              "value": {
+                "description": "The value returned from the tool.",
+                "oneOf": [
+                  {
+                    "type": "object",
+                    "additionalProperties": true
+                  },
+                  {
+                    "type": "number"
+                  },
+                  {
+                    "type": "string"
+                  },
+                  {
+                    "type": "boolean"
+                  }
+                ]
+              }
+            },
+            "not": { "required": ["error"] }
+          },
+          {
+            "properties": {
+              "success": { "enum": [false] },
+              "error": {
+                "type": "object",
```

Comment:
Could be nice to define this object as a ref w/ a definition of "ToolError"

Reply:
👍 done!